byterun/startup_aux.c

```diff
@@ -106,7 +106,7 @@ void caml_parse_ocamlrunparam(void)
       case _T('W'): scanmult (opt, &caml_runtime_warnings); break;
       }
       while (*opt != _T('\0')){
-        if (*opt++ == ',') break;
+        if (*opt++ == _T(',')) break;
```

Comment:
I think this is an unrelated bug in #1200? It should really go in a separate PR.

Reply:
Just a reminder that this does need to go in a PR, though! Have you pushed a branch for it?

Reply:
Will do it shortly, thanks.